src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesCornerTests.cs

```diff
@@ -324,5 +324,115 @@ public static void ZeroLengthTransformPKCS7()
                 }
             }
         }
+
+#if netstandard17
```

Comment:
Since these tests are focused highly on the padding (instead of other AES-ness) these could make sense in a different file.  We've just frequently (in crypto) had an obvious "this belongs with these other tests, and we wouldn't have thought of putting it in a different file except for the 1.6 test build".

I'm fine with whichever way.